featurebyte/service/offline_store_feature_table_manager.py

```diff
@@ -482,6 +530,84 @@ async def _create_or_update_feast_registry(
         assert output is not None
         return output
 
+    async def _update_precomputed_lookup_feature_tables_no_op_enable(
+        self,
+        feature_lists: List[FeatureListModel],
+        feature_store_model: FeatureStoreModel,
+    ) -> None:
+        # Called when online enabling, but no features are enabled but a new deployment is made on
+        # an existing feature list
+        async for (
+            feature_table_dict
+        ) in self.offline_store_feature_table_service.list_documents_as_dict_iterator(
+            query_filter={"precomputed_lookup_feature_table_info": None},
+            projection={"_id": 1, "feature_ids": 1},
+        ):
+            feature_ids = feature_table_dict["feature_ids"]
+            await self._update_precomputed_lookup_feature_tables(
+                feature_table_dict["_id"],
+                feature_ids,
+                feature_lists,
+                feature_store_model,
+            )
```

Comment:
The main reason of doing this is because of new relationship that may not be covered in existing pre-computed lookup feature tables, right?

Reply:
It should be a new feature list, right? If existing feature list, the relationship should be freeze during feature list construction.

Reply:
Yes, this is to handle the new feature list that is being online enabled.